### PR TITLE
Align nullability of BO Vertrag and COM Vertragsteil with BO4E.NET

### PR DESCRIPTION
**Pull request by hf-kklein**
as in BO4E.net v0.4.2
https://github.com/Hochfrequenz/BO4E-dotnet/releases/tag/0.4.2

https://github.com/Hochfrequenz/BO4E-dotnet/blob/c7c9525849cb8c29c5f1d57c53e3435cd281ba43/BO4E/COM/Vertragsteil.cs#L19
https://github.com/Hochfrequenz/BO4E-dotnet/blob/c7c9525849cb8c29c5f1d57c53e3435cd281ba43/BO4E/BO/Vertrag.cs

### CODE REVIEW

**Comment by hf-kklein** [comment]
obsoletes #144

File: bo/businessobject.go [hunk 1]
```diff
@@ -99,8 +99,8 @@ func NewBusinessObject(typ botyp.BOTyp) BusinessObject {
 		bo = new(Vertrag)
 		bo.(*Vertrag).BoTyp = typ
 		bo.(*Vertrag).VersionStruktur = defaultVersionStruktur
-		bo.(*Vertrag).Vertragspartner1.BoTyp = botyp.GESCHAEFTSPARTNER
-		bo.(*Vertrag).Vertragspartner2.BoTyp = botyp.GESCHAEFTSPARTNER
+		// bo.(*Vertrag).Vertragspartner1.BoTyp = botyp.GESCHAEFTSPARTNER
```
Comment (GodsBoss) [inline bo/businessobject.go:102]:
```suggestion
```

File: bo/businessobject.go [hunk 2]
```diff
@@ -99,8 +99,8 @@ func NewBusinessObject(typ botyp.BOTyp) BusinessObject {
...
+		// bo.(*Vertrag).Vertragspartner2.BoTyp = botyp.GESCHAEFTSPARTNER
```
Comment (GodsBoss) [inline bo/businessobject.go:103]:
```suggestion
```

**Review by GodsBoss — CHANGES_REQUESTED** [review]
I'd prefer not to leave outcommented code, the rest looks fine to me.

**Review by GodsBoss — APPROVED** [review]

**Pull request merged**